src/Moose-Core/MooseModel.class.st

```diff
@@ -369,6 +369,23 @@ MooseModel >> exportToMSEStream: aStream [
 	self class export: self to: aStream
 ]
 
+{ #category : #actions }
+MooseModel >> flush [
+	"Clean the cache of all the entities of the model"
```

Comment:
```suggestion
	"Clean the private state caches of all the entities of the model. This will not impact attributes in the private state (informations you cannot derive)"
```